launcher/core/injector/windllinjector.cpp

```diff
@@ -204,7 +204,8 @@ void WinDllInjector::processExited(int exitCode)
 QString WinDllInjector::fixProbeDllPath(const QString &probeDll)
 {
     QString dllPath = QDir::toNativeSeparators(probeDll);
-    dllPath.replace(QLatin1String("gammaray_probe"), QLatin1String("gammaray_probe_win"));
```

Comment:
if you replace `gammaray_probe` with `gammaray_probe_win` it works for me

Reply:
why do we allow two different plugin layouts ....

Reply:
I just realize - this makes the whole replace a no-op, so the whole replace line can be removed?! I still don't know what you mean by different plugin layouts.

Reply:
nvm the above comment, it isn't a noop:

```
dllPath.replace(QLatin1String("gammaray_probe_win"), QLatin1String("gammaray_winloader"));
```

Reply:
With that fix in, both the release build and the debug build work for me. Most notably, the debug probe build as part of the release build also works.